### ISSUE TITLE
I owe you money

### ISSUE THREAD

**Issue by kangarko** [Puharesource/TitleManager#429]
Hi Tarkan,

Last year you've invited some friends into my training program and after going through our database after migrating our site, I found you have outstanding commission ready to be paid.

What's your paypal account? Or you want me to make a Kofi donation instead?

I am sorry for opening an issue here, but I wanted you to see this just in case my email goes to spam!

You can delete this issue afterwards, thank you!

-Matej

**Comment by kangarko** [comment]
Or I can buy you a two-year gold sub on Ko-fi if you want?

**Comment by Puharesource** [comment]
Hey Martej,

I was unable to get any of the payouts that I requested through the platform, so I gave up.
Just send it to my paypal at: pppmtarkan@gmail.com

Thanks,
Tarkan

**Comment by kangarko** [comment]
Sent you 75 Euro (almost 600 DKK) - Merry Christmas!

Do you remember what went wrong? We have a 30-day waiting period before you can get paid in case someone asks for a refund. I'm going to check this for you.

Matej

**Comment by Puharesource** [comment]
Thank you and Merry Christmas to you too!

The dashboard was a little weird, sometimes I saw people had purchased the course and during the 30 day waiting period they'd just disappear. Other times I did wait for the 30 day period to be over and I believe it would just not allow me to get a payout for whatever reason that I can't remember anymore.

**Comment by kangarko** [comment]
We have removed people who asked for a refund but typically that is under 6%, it's been a while so I can't remember either. 

I've rebuilt our portal such that even if we delete the user internally, they will just get rendered as "Deleted" instead of the row being removed completely, making it look like a scam haha! That was a failure and I apology.

You're welcome to rejoin again — I've given you full access at your personal email, we also have new banners and invite links.
